### PR TITLE
Added Date and Time picker in DialogsActivity.kt

### LINKED ISSUE #141: Add Material Date Picker

### LINKED ISSUE #142: Add Material Time Picker

### PR DESCRIPTION
**Pull request by yogeshpaliyal**
fix #142 , fix #141

### CODE REVIEW

File: app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt [hunk 1]
```diff
@@ -284,6 +308,37 @@ fun ShowDialog(type: DialogType, onDismiss: () -> Unit) {
                 onDismissRequest = onDismiss,
                 shape = RoundedCornerShape(16.dp)
             )
+        DialogType.DATEPICKER -> {
+            val context = LocalContext.current
+            (context as? FragmentActivity)?.supportFragmentManager?.let { manager ->
+
+                val builder = MaterialDatePicker.Builder.datePicker()
```
Comment (Gurupreet) [inline app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: I am not sure it's Compose component. Hopefully we will have native compose component for date and time picker.

**Review by Gurupreet — APPROVED** [review]
Great 💯

File: app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt (hunk 1 above)
Comment (yogeshpaliyal) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Yes, but I don't found composable for this

Comment (yogeshpaliyal) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Still there is no material design's date picker for compose

Comment (Gurupreet) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Yup so far no :/

Comment (yogeshpaliyal) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Till then we can use this, then we can replace with composable.

Comment (Gurupreet) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Yes definitely, thanks for adding :)

Comment (yogeshpaliyal) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Is there any material theme adapter that can create theme for AndroidView

Comment (Gurupreet) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: There is [Theme Adapter](https://material-components.github.io/material-components-android-compose-theme-adapter/) But it's for exporting existing themes in Compose.

Comment (yogeshpaliyal) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Yes, but opposite of that.
Like if i want to customise DatePicker or TimePicker we have to define XML styling for those.

Comment (Gurupreet) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Yup not that I am aware of yet. But let me dig haha

Comment (yogeshpaliyal) [reply app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt:315]: Sure, I will also search for that.

**Pull request merged**